packages/fe-fpm-writer/test/integration/__snapshots__/basic-lrop-app.test.ts.snap

```diff
@@ -54,6 +54,52 @@ Object {
         }
     };
 });
+",
+    "state": "modified",
+  },
+  "js/webapp/ext/myCustomHeaderSection/MyCustomHeaderSection.fragment.xml": Object {
+    "contents": "<core:FragmentDefinition xmlns:core=\\"sap.ui.core\\" xmlns=\\"sap.m\\" xmlns:macros=\\"sap.fe.macros\\">
+	<VBox>
+		<Button core:require=\\"{ handler: 'v4travel/ext/myCustomHeaderSection/MyCustomHeaderSection'}\\" text=\\"MyCustomHeaderSection\\" press=\\"handler.onPress\\" />
+	</VBox>
+</core:FragmentDefinition>",
+    "state": "modified",
+  },
+  "js/webapp/ext/myCustomHeaderSection/MyCustomHeaderSection.js": Object {
+    "contents": "sap.ui.define([
+    \\"sap/m/MessageToast\\"
+], function(MessageToast) {
+    'use strict';
+
+    return {
+        onPress: function(oEvent) {
+            MessageToast.show(\\"Custom handler invoked.\\");
+        }
+    };
+});
+",
+    "state": "modified",
+  },
+  "js/webapp/ext/myCustomHeaderSectionEdit/MyCustomHeaderSectionEdit.fragment.xml": Object {
+    "contents": "<core:FragmentDefinition xmlns:core=\\"sap.ui.core\\" xmlns=\\"sap.m\\">
+	<VBox>
+		<Input core:require=\\"{ handler: 'v4travel/ext/myCustomHeaderSectionEdit/MyCustomHeaderSectionEdit'}\\" label=\\"MyCustomHeaderSectionEdit\\" change=\\"handler.onPress\\" />
+	</VBox>
+</core:FragmentDefinition>",
+    "state": "modified",
+  },
+  "js/webapp/ext/myCustomHeaderSectionEdit/MyCustomHeaderSectionEdit.js": Object {
+    "contents": "sap.ui.define([
+    \\"sap/m/MessageToast\\"
+], function(MessageToast) {
+    'use strict';
+
+    return {
+        onPress: function(oEvent) {
```

Comment:
could we please rename `onPress` to `onChange` if input is used and event is `change`?

Reply:
You're right! Changed to `onChange`.